### PR TITLE
Bug 1403974: Improving styling on "ASAP" button to make it clear that it is not the default

### PR DESCRIPTION
**Pull request by collin5**
- Changes the color of ASAP button to non default.
- Adds alert with info on usage of ASAP button.

### CODE REVIEW

File: ui/app/templates/release_scheduled_change_modal.html [hunk 1]
```diff
@@ -6,14 +6,21 @@ <h3 class="modal-title" ng-show="sc.name && sc.when">Edit Scheduled Change for {
...
     <h4>Scheduled Change Details</h4>
-
       <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+            <div class="help-block">
+                <div ng-hide="hideAsapInfo" class="alert alert-info" auto-hide="3000">
+                    <a href="#" class="close" arial-label="close" ng-click="hideAsapInfo=true">&times;</a>
+                    <strong>Info !</strong> Use ASAP to make changes as soon as possible
+            </div>
+        </div>
```
Comment (mozbhearsum) [inline ui/app/templates/release_scheduled_change_modal.html]: I don't think this notification in particularly useful. "ASAP" is a pretty well understood term already, so an additional explanation isn't really helpful.

File: ui/app/templates/release_scheduled_change_modal.html [hunk 2]
```diff
@@ -6,14 +6,21 @@ <h3 class="modal-title" ng-show="sc.name && sc.when">Edit Scheduled Change for {
...
       <div class="col-md-8 col-md-offset-2">
         <label for="id_when">When (local time)</label>
+      </div>
         <div class="form-group dropdown" is-open="calendar_is_open" ng-class="{'has-error': errors.when}">
           <div class="input-group">
             <span class="input-group-btn">
-              <label class="btn btn-default" ng-click="toggleAutoTime()" id="btn__auto-time"><input type="checkbox" ng-show="false" ng-model="auto_time" auto-complete="off">ASAP</label>
+              <label class="btn btn-primary" ng-click="toggleAutoTime()" id="btn__auto-time"><input type="checkbox" ng-show="false" ng-model="auto_time" auto-complete="off">ASAP</label>
```
Comment (mozbhearsum) [inline ui/app/templates/release_scheduled_change_modal.html]: This styling change doesn't really improve anything...the button still has an unclear state. I'm not exactly sure what it should look like though...

Comment (collin5) [reply ui/app/templates/release_scheduled_change_modal.html]: @mozbhearsum  Hmm, will us adding some color and a border to it do the trick ?
i.e
screen shot 2017-10-19 at 11 51 10 am

Comment (mozbhearsum) [reply ui/app/templates/release_scheduled_change_modal.html]: The "after" screenshot looks like the button has already been depressed still.

Comment (collin5) [reply ui/app/templates/release_scheduled_change_modal.html]: @mozbhearsum What about adding a little elevation to it, to something like :

screen shot 2017-10-23 at 10 30 55 am

Comment (mozbhearsum) [reply ui/app/templates/release_scheduled_change_modal.html]: @kmoir - what do you think? If I remember correctly, you found the current styling of the ASAP button a bit unclear.

**Comment by collin5** [comment]
Hi, @mozbhearsum  I made some changes to make the button more intuitive but also appear smoother than the previous design. [ inspired by GitHub buttons ] What do you think?

screen shot 2017-11-09 at 3 45 53 pm

**Comment by collin5** [comment]
Awesome, thanks 👍

**Pull request merged**